### PR TITLE
feat(config): add support for Inovelli LZW45 with partial parameters

### PR DESCRIPTION
**Pull request by Petro31**
Adds support for the recently released Inovelli LZW45 LED Light Strip. This was imported from the OZW database.

closes: https://github.com/zwave-js/node-zwave-js/pull/1525

### CODE REVIEW

File: packages/config/config/devices/0x031e/lzw45.json [hunk 1]
```diff
@@ -0,0 +1,1169 @@
...
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic_Set",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json]:
```suggestion
			"label": "Basic Set",
```

File: packages/config/config/devices/0x031e/lzw45.json [hunk 2]
```diff
@@ -0,0 +1,1169 @@
...
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Switch_MultiLevel_Set",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json]:
```suggestion
			"label": "Switch Multilevel Set",
```

File: packages/config/config/devices/0x031e/lzw45.json [hunk 3]
```diff
@@ -0,0 +1,1169 @@
...
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Switch_MultiLevel_Start/Stop",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json]:
```suggestion
			"label": "Switch Multilevel Start/Stop",
```

File: packages/config/config/devices/0x031e/lzw45.json [hunk 4]
```diff
@@ -0,0 +1,1169 @@
...
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Number of Pixels",
+			"description": "When individually addressable LEDs are used, this parameter tells the controller the number of pixels that are attached.  0: Automatic recognition of pixels. Range 1 - 130: Set the fixed value of the pixel bit. Default: 0",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json]: The description should not have default values and ranges and such. Special meanings (e.g. off) should be conveyed with an `"option"` where it makes sense. Same goes for the other params.

File: packages/config/config/devices/0x031e/lzw45.json [hunk 5]
```diff
@@ -0,0 +1,1169 @@
...
+			"defaultValue": 3,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Ramp Rate",
+			"description": "This changes the speed in which the light strip turns on or off. For example, when a user sends the switch a basicSet(value: 0xFF) or basicSet(value: 0x00), this is the speed in which those actions take place. A setting of 0 should turn the light immediately on or off (almost like an on/off switch). Increasing the value should slow down the transition speed. A setting of 99 should keep this in sync with parameter 2. Range 0 - 98: Set the Ramp Rate, 99: Keep in sync with parameter. Default: 99",
+			"valueSize": 1,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json:63]: does this have a unit?

File: packages/config/config/devices/0x031e/lzw45.json [hunk 6]
```diff
@@ -0,0 +1,1169 @@
...
+			"maxValue": 99,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"9": {
+			"label": "Default Color",
+			"description": "Byte(3-2): Values between 2700-6500 represent a color temperature. Byte(1-0): Values between 0-361, represent the color on the Hue color wheel. The value of 361 represents a random color. Range: 0 - 425984362 Default: 262144000",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json]: That should be a partial param

File: packages/config/config/devices/0x031e/lzw45.json [hunk 7]
```diff
@@ -0,0 +1,1169 @@
...
+			"maxValue": 100,
+			"defaultValue": 10,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"18": {
+			"label": "Periodic Power & Energy Reports",
+			"description": "Time period between consecutive power & energy reports being sent (in seconds). The timer is reset after each report is sent. Range: 0, 30 - 32767 Default: 3600",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json]: unit please

File: packages/config/config/devices/0x031e/lzw45.json [hunk 8]
```diff
@@ -0,0 +1,1169 @@
...
+			"defaultValue": 3600,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"19": {
+			"label": "Energy Reports",
+			"description": "The energy level change that will result in a new energy report being sent (in kWh)\n\t  0 = 0.01kWh, 10 = 0.1kWh, 100 = 1kWh. Range:0-127 Default: 10",
+			"valueSize": 1,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json:210]: `"unit": "0.01 kWh"`

File: packages/config/config/devices/0x031e/lzw45.json [hunk 9]
```diff
@@ -0,0 +1,1169 @@
...
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"21[0x7f00]": {
+			"label": "Quick Strip Effect Intensity",
+			"description": "Intensity of the LED strip.  Course: 0 - 10, where 10 = 100%.  Fine: 0-99, where 99 = 100%.  Default: 0",
+			"valueSize": 4,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json:235]: Do you mean "coarse"?

File: packages/config/config/devices/0x031e/lzw45.json [hunk 10]
```diff
@@ -0,0 +1,1169 @@
...
+					"label": "Slow Blink",
+					"value": 4
+				},
+				{
+					"label": "Fast Fade",
+					"value": 5
+				},
+				{
+					"label": "Slow Fade",
+					"value": 5
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json]: value duplicated

File: packages/config/config/devices/0x031e/lzw45.json [hunk 11]
```diff
@@ -0,0 +1,1169 @@
...
+				}
+			]
+		},
+		"21[0x40000000]": {
+			"label": "Quick Strip Effect: Color Type",
+			"description": "Color type. 0 = Hue Color Wheel, 1 = Color Temp. Default: 0",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json]: remove

File: packages/config/config/devices/0x031e/lzw45.json [hunk 12]
```diff
@@ -0,0 +1,1169 @@
...
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"21[0x3f000000]": {
+			"label": "Quick Strip Effect: Effect",
+			"description": "The light LED strip effect. 0 = Off, 1 = Solid, 2 = Chase, 3 = Fast Blink, 4 = Slow Blink, 5 = Fast Fade, 6 = Slow Fade.  Default: 0",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json]: remove

File: packages/config/config/devices/0x031e/lzw45.json [hunk 13]
```diff
@@ -0,0 +1,1169 @@
...
+				},
+				{
+					"label": "Color Temp",
+					"value": 1
+				}
+			]
+		},
+		"22[0x7]": {
+			"label": "Custom Strip Effect: Action 1 Effect",
+			"description": "This is the effect of Action 1.  0 = Fade, 1 = Fade Blend, 2 = Flash, 3 = Chase, 4 = Chase Blend.  Default: 0",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json]: remove

File: packages/config/config/devices/0x031e/lzw45.json [hunk 14]
```diff
@@ -0,0 +1,1169 @@
...
+				},
+				{
+					"label": "Chase Blend",
+					"value": 4
+				}
+			]
+		},
+		"22[0xf8]": {
+			"label": "Custom Strip Effect: Action 1 Color",
+			"description": "This is the color of Action 1.  0 = Off, 1 = 2700K, 2 = 4500K, 3 = 6500K, 4 = Red, 5 = Orange, 6 = Yellow, 7 = Yellow Green, 8 = Green, 9 = Spring Green, 10 = Cyan, 11 = Azure, 12 = Blue, 13 = Violet, 14 = Magenta, 15 = Random.  Default: 0",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw45.json]: remove (same goes for the other descriptions that duplicate the other info)

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Looks pretty good. Some of the descriptions are unnecessary and should be removed. And btw. these are some **nasty** partial params :D

File: packages/config/config/devices/0x031e/lzw45.json (hunk 9 above)
Comment (Petro31) [reply packages/config/config/devices/0x031e/lzw45.json:235]: 😆 yes

File: packages/config/config/devices/0x031e/lzw45.json (hunk 12 above)
Comment (Petro31) [reply packages/config/config/devices/0x031e/lzw45.json]: remove what?  The whole description or the values?

Comment (Petro31) [reply packages/config/config/devices/0x031e/lzw45.json]: I'm going to assume you mean the 0 = off etc.

File: packages/config/config/devices/0x031e/lzw45.json (hunk 5 above)
Comment (Petro31) [reply packages/config/config/devices/0x031e/lzw45.json:63]: It does not have a unit.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Doesn't look like all of my change requests made it into your recent commits

**Comment by Petro31** [comment]
> Doesn't look like all of my change requests made it into your recent commits

Ugh, yeah, I didn't see the collapsed comments!

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**